omod-1.10/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_10/OrderController1_10Test.java

```diff
@@ -485,7 +486,12 @@ public void shouldGetAllInActiveOrdersForAPatient() throws Exception {
 		        PropertyUtils.getProperty(resultList.get(1), "uuid").toString() });
 		assertThat(uuids, hasItems(expectedOrderUuids));
 	}
-	
+	@Test
+	public void shouldReturnExceptionIfNoUuidIsSpecified() throws Exception {
```

Comment:
Can you improve the name of the test? eg. `shouldThrowExceptionIfPatientQueryParameterNotSpecified`

Reply:
Ok thanks alot for that revision.

Reply:
Ok thanks alot for that revision.